.github/workflows/ci.yml

```diff
@@ -0,0 +1,23 @@
+name: test
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+jobs:
+  test:
+    strategy:
+      fail-fast: false
+      matrix:
+        java: [8, 11, 17-ea]
+        scala: [2.13.6, 3.0.1]
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - uses: coursier/cache-action@v6
+    - uses: actions/setup-java@v2
+      with:
+        distribution: adopt
+        java-version: ${{matrix.java}}
+    - name: Test
+      run: sbt ++${{matrix.scala}} test core/headerCheck package
```

Comment:
Not sure if that matters, but we used to do a `publishLocal` instead of a `package`

Reply:
Yeah idk... lots of repos don't even go as far as `package` but just stop at `test`.... I think I'm going to leave it like this.